KsApi/models/graphql/adapters/DeletePaymentSourceEnvelope+DeletePaymentSourceMutation.Data.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import ReactiveSwift
+
+extension DeletePaymentMethodEnvelope {
+  static func from(_ data: GraphAPI.DeletePaymentSourceMutation.Data) -> DeletePaymentMethodEnvelope? {
+    guard let paymentSourceDelete = data.paymentSourceDelete else {
+      return nil
+    }
+
+    guard let storedCards = paymentSourceDelete.user?.storedCards?.fragments.userStoredCardsFragment.nodes
+    else {
+      return DeletePaymentMethodEnvelope(storedCards: [])
```

Comment:
Probably want to test this scenario too.

Reply:
Side question, could this guard and the one above it be consolidated and instead of returning nil you return empty? Does that fit the use case/expected behavior on the UI?

Reply:
yea that's a good point, I just realized that returning a successful mutation with no cards deleted should be a front end error. Because there's no scenario where we delete a payment method successfully and no payment method gets deleted on the backend.